package.json

```diff
@@ -1,13 +1,13 @@
 {
-	"name": "@silverorange/swat",
-	"version": "0.1.0",
-	"private": true,
-	"scripts": {
-		"check-if-pretty": "prettier --list-different '{Swat,SwatDB,SwatI18N}/**/*.php'",
-		"make-pretty": "prettier --write '{Swat,SwatDB,SwatI18N}/**/*.php'"
-	},
-	"devDependencies": {
-		"@prettier/plugin-php": "^0.19.0",
-		"prettier": "^2.8.3"
-	}
+  "name": "@silverorange/swat",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "prettier": "prettier --list-different .",
```

Comment:
Prettier was configured to intentionally only apply formatting to specific files.

Reply:
Yes, it was set to `prettier --list-different '{Swat,SwatDB,SwatI18N}/**/*.php'`.  But I switched to using `php-cs-fixer` to handle the code style for PHP files, so prettier doesn't do that anymore.

However, I have prettier working on all the other files that it (natively) checks (JS/TS, CSS/LESS/SASS, HTML, YAML, JSON, MD), so we get the benefit of a consistent style (and some simple checking) on those files now (e.g. it did catch a duplicate entry in `dependencies/swat.yaml`, and I believe a weird issue in one of the CSS files).